imports/startup/server/fixtures.js

```diff
@@ -319,3 +278,38 @@ function generateRandomPairing(users) {
   }
   return pairing;
 }
+
+/** Generate an array of mocked users */
+function generateUsers(num, roles) {
+  return Array(num).fill().map((_, i) => {
+    var testNum = 'test' + i.toString()
+    return {
+      email: testNum + '@test.com',
+      password: 'password',
+      profile: {
+        fullName: testNum
+      },
+      task: testNum + ' task',
+      roleTitle: roles.Undergrad,
+      isAdmin: false
+    }
+  })
+}
+
+/** Generate random affinities between each pair of users (non-symmetric) */
+function generateAffinities(userIds, groupId) {
+  const affinityVals = [-1, 0, 0.33, 0.66, 1]
+
+  affinitiesPerUser = userIds.map((me) => {
+    return userIds.filter((id) => id !== me).map((them) => {
+      return {
+        helperId: me,
+        helpeeId: them,
+        groupId: groupId,
+        value: affinityVals[Math.floor(Math.random()*affinityVals.length)]
```

Comment:
I do believe lodash is imported here so you can just do `_.sample(affinityVals)`